docs/Software Architecture Document/Component Interaction View/assets/plugin_manager_interaction_view.puml

```diff
@@ -0,0 +1,32 @@
+@startuml
```

Comment:
Please, use [the same style](https://github.com/smartdevicelink/sdl_core_guides/blob/master/docs/Software%20Architecture%20Document/Component%20Interaction%20View/assets/app_launch.plantuml#L27)